### PR TITLE
Eytecz mod LGX Lite

### PR DESCRIPTION
**Pull request by Eytecz**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

**Comment by Eytecz** [comment]
Which mods/configurations/profiles are added by this PR? LGX Lite Stealthburnwr CW2 style mount.
How was it tested? Tested since beta, this is the updated version to the released files from the CW2 and Stealthburner
Any background context you want to provide?
Screenshots (if appropriate)
Further notes

**Comment by newphreak** [comment]
You have to PR the whole mod, not just link to it

**Comment by Eytecz** [comment]
Files have been added.

**Comment by Eytecz** [comment]
Can this mod be added to the VoronUsers Printer Mods now? :)

File: printer_mods/Eytecz/LGX_Lite_Stealthburner_CW2_style_mount/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+LGX Lite extruder mount for the Voron Stealthburner. The standard bowden clip delivered with the LGX Lite is to be removed, and a standard UM2 connector has to be inserted in the top printed part. This style clip is to be used: https://nl.aliexpress.com/item/32896103430.html. If you're lucky you might have some laying around already.
+
+# LGX Lite mount ERCF
+The ERCF version is made to accomodate the LGX Lite with the stealthburner and the new carriage, including the toolhead sensor as it is being used for the ERCF. The housing style is matching with the original CW2 design. I recommend to insert a 3mm long 4x2mm bowden tube on the bottom of the LGX Lite to guide the filament properly. The cover and cable carrier mount can be re-used from the original version. The top of the front body is made to insert the bowden tube until it reaches into the LGX Lite, through a standard bowden tube clip with retainer ring. A lever latch is designed to change the pretension on the filament. 
+
+![2022-08-10 21_20_59-Solid Edge 2022 - Assembly -  LGX_Lite_Stealthburner_ERCF asm](https://user-images.githubusercontent.com/94404962/184004932-a4767190-644e-4b3a-a60f-1d7e687b40dd.png)
```
Comment (MapleLeafMakers) [inline printer_mods/Eytecz/LGX_Lite_Stealthburner_CW2_style_mount/README.md:7]: These images urls should be relative to the current folder, for example:
`2022-08-10 21_20_59-Solid Edge 2022 - Assembly -  LGX_Lite_Stealthburner_ERCF asm`

Comment (Eytecz) [reply printer_mods/Eytecz/LGX_Lite_Stealthburner_CW2_style_mount/README.md]: Hi,

I have just updated these links. I believe everything else should be correct now?

Comment (MapleLeafMakers) [reply printer_mods/Eytecz/LGX_Lite_Stealthburner_CW2_style_mount/README.md]: You've only changed the descriptions, not the links.  The descriptions were already fine, the links were not.  They should be relative links pointing to files within this repository. For example  you have `https://user-images.githubusercontent.com/94404962/184004932-a4767190-644e-4b3a-a60f-1d7e687b40dd.png` which is actually pointing to your IMG_01.png file, so the link should be `./LGX_Lite_Stealthburner_ERCF/Images/IMG_01.png`

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**